config/env.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
```

Comment:
This and many other files were not written by hand by me. They were always part of the project but were hidden under `node_modules` from [CRA](https://create-react-app.dev/docs/getting-started/). I ran `npm eject` so the files are visible/editable and we can customize for our needs